packages/flutter_tools/lib/src/usage.dart

```diff
@@ -75,9 +75,15 @@ class Usage {
   /// reports coming from the same computer.
   String get clientId => _analytics.clientId;
 
-  void sendCommand(String command) {
-    if (!suppressAnalytics)
-      _analytics.sendScreenView(command);
+  void sendCommand(String command, { Map<String, String> parameters }) {
+    if (suppressAnalytics)
+      return;
+
+    if (parameters != null) {
+      parameters.forEach(_analytics.setSessionValue);
```

Comment:
we want to report cd3 and cd4 on every "hit", because they are hit-level dimensions.